gdg/__init__.py

```diff
@@ -144,6 +144,17 @@ def filename_lev(a, f):
     else:
         return levenshtein(a, fn.group(1))
 
+# Returns a list of key-value pairs {image, distance}
+def filter_images_by_lev(name, image_list, max_dist):
+    if max_dist == -1:
+        yield return map(lambda x: { 'image': x, 'distance': 0 }, image_list)
```

Comment:
:neutral_face: You can't have `return x` in a generator function.  In order to do this, foreach over the image list and yield a new kvp, then return with no arguments.

Testing your code before pushing is highly recommended :smirk: